coins.yml

```diff
@@ -210,81 +213,93 @@
   handle: bitcoin
   name: Bitcoin
   decimals: 8
+  blockTime: 600000
```

Comment:
can we do `10000` - this means it will check every 5-10 seconds for the new transactions

Reply:
No new blocks, seems we need to check mempool

Reply:
I don't think we need an infra in place for it, currently it just checks confirmed blocks, we can add this logic in the future version